dataprofiler/tests/profilers/test_profiler_class_options.py

```diff
@@ -0,0 +1,110 @@
+from dataprofiler.profilers.profiler_options import ProfilerOptions
+from dataprofiler.tests.profilers.test_base_option import TestBaseOption
+
+
+class TestProfilerOptions(TestBaseOption):
+    
+    option_class = ProfilerOptions
+    keys = ["structured_options"]
+    
+    @classmethod
+    def get_options(self, **params):
+        options = ProfilerOptions()
+        options.set(params)
+        return options
+    
+    def test_init(self):
+        options = self.get_options()
+        for key in self.keys:
+            self.assertTrue(key in options.properties)
```

Comment:
for this, can do `self.assertIn(key, options.properties)`
will have better error reporting

Reply:
Makes sense, modified in profiler, structured, and numerical options tests